kod/util/system.kod

```diff
@@ -1757,7 +1757,13 @@ messages:
          {
             Send(poNecromancerBalance,@IncrementHKCounter);
          }
+
+		   RecordStat(STAT_PLAYERDEATH, Send(what,@GetTrueName), Send(killer,@GetTrueName), Send(oRoom,@GetName), Send(stroke_obj,@GetName), TRUE);	 
       }
+		else
```

Comment:
The STAT_PLAYERDEATH RecordStat() call may need to deal with a $ room name and $ stroke_obj. At the moment it looks like RecordStat() will log an error for these, is it possible instead to record the data in the database without these fields if they aren't present?

The only cause of $ room I can think of right now is a necromancer dying while offline. There are several ways of dying without having stroke_obj sent - I think dying at forest shrine will do this. We should fix all those cases eventually but I think it's best that RecordStat deal with those potential cases instead of logging an error with the C call.